content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm

```diff
@@ -617,7 +627,7 @@ button.btn.btn-default.multiselect-clear-filter{
 							</th>
 							<td>
 								<div class="form-group">
-									<a class="textPanelFooter" href="#fileURLHolder" onclick="document.getElementById('fileURLHolder').style.overflow='auto';document.getElementById('fileURLHolder').select();document.execCommand('copy');return false;">$tlang.getString("props.select")</a>
+									<a class="textPanelFooter" href="#fileURLHolder" onclick="copyToClipboard();">$tlang.getString("props.select")</a>
 									<a class="textPanelFooter" href="$model.accessUrl" target ="_blank">$tlang.getString("props.open")</a>
```

Comment:
I'd make this so that you attach the handler from the js, not using an onclick attribute. Just add a document ready right after your function body. This helps us move JS out of of the templates down the line.

Reply:
Good